File: tests/indexing/parsing/lang/fixtures/java/NestedBlocks.java
class Nesting {

    int process(int[] xs, int threshold) {
        int total = 0;
        for (int x : xs) {
            if (x > threshold) {
                try {
                    total = total + 100 / x;
                } catch (ArithmeticException e) {
                    total = total - 1;
                }
            }
        }
        return total;
    }
}

File: tests/indexing/parsing/lang/fixtures/kotlin/NestedBlocks.kt
class Nesting {

    fun process(xs: IntArray, threshold: Int): Int {
        var total = 0
        for (x in xs) {
            if (x > threshold) {
                try {
                    total = total + 100 / x
                } catch (e: ArithmeticException) {
                    total = total - 1
                }
            }
        }
        return total
    }
}
